todo_app/app.py

```diff
@@ -2,17 +2,30 @@
 
 from todo_app.flask_config import Config
 from todo_app.data.session_items import get_items,add_item
+from todo_app.data.trello_helper import TrelloHelper
+
+import os
+
 app = Flask(__name__)
 app.config.from_object(Config())
 
+helper = TrelloHelper()
 
 @app.route('/')
 def index():
-    items = get_items()
-    return render_template('index.html', items=items)
+    helper.refresh_lists()
+    helper.set_list_ids()
```

Comment:
Do you want to refresh the lists every time - do you expect them to meaningfully change?

This bakes in a solution to a potential issue, but it seems unlikely that we expect to be changing those ids on the fly, and does add an overhead to the speed of the page load.

If we _do_ expect these to change, arguably we should also be refreshing these in each of the move-status functions too, but personally I'd lean towards the simpler approach of moving these calls into the constructor (`__init__` function) of the TrelloHelper class so that they're automatically called on first creation

That also has the advantage that a niche set of circumstances here could currently cause an error!

* User loads the homepage
* We release a change to the app, such that a new version of the code spins up, which hasn't yet had anyone hit the homepage
* Request comes to "complete" a task from the user who had already loaded the homepage
* The new app hasn't yet loaded and set the list ids, so it errors

While that seems rare and not a huge cost, this particular structure of having some initial set up take place outside the constructor is a common pattern that can easily bite you when changes happen later that assume a particular order of loading has happened